openapi/reputation/reputation.yaml

```diff
@@ -218,6 +218,14 @@ components:
             url:
               description: The URL to the review
               type: string
+            modifiedAt:
+              description: Date of modified
```

Comment:
Oh I guess maybe this is the PR to comment on. This could use some more details like `Date when the system last updated this entity`

Reply:
I agree with your suggestion.